src/chrome/content/rules/Linode.xml

```diff
@@ -1,32 +1,67 @@
 <!--
-	Problematic hosts in *linode.com:
+	Nonfunctional hosts in *linode.com:
+
+		- speedtest.atlanta ¹
+		- speedtest.dallas ¹
+		- speedtest.frankfurt ¹
+		- speedtest.fremont ¹
+		- speedtest.london ¹
+		- speedtest.newark ¹
+		- speedtest.singapore ¹
+		- speedtest.tokyo ¹
 
-		- status *
+		- mirrors ¹
+		- atlanta.mirrors ¹
+		- dallas.mirrors ¹
+		- frankfurt.mirrors ¹
+		- fremont.mirrors ¹
+		- london.mirrors ¹
+		- newark.mirrors ¹
+		- singapore.mirrors ¹
+		- tokyo.mirrors ¹
 
-	* StatusPage.io
+		- *.members ²
+		- *.nodebalancer ²
 
+	¹ Refused
+	² Customer load balancers and VPSes. Mismatched, refused or time out.
 
-	Fully covered hosts in *linode.com:
+	Problematic hosts in *linode.com:
 
-		- (www.)?
-		- blog
-		- forum
-		- library
-		- manager
-		- stats
-		- status	(→ linode.statuspage.io)
+		- apt ¹
+		- status ²
 
+	¹ "gnutls_handshake() warning: The server name sent was not recognized"
+	  SSL Labs says "Incorrect SNI alerts: apt.linode.com"
+	  It works in Chrome and Firefox, though.
```

Comment:
For what it's worth, Apache 2.2.30 and 2.4.10 no longer send the warning. ([Bug 56241](https://bz.apache.org/bugzilla/show_bug.cgi?id=56241)) https://apt.linode.com/ is running 2.2.22.